transmart-core-api/src/main/groovy/org/transmartproject/core/multidimquery/NumericalValueAggregates.groovy

```diff
@@ -1,11 +1,12 @@
 package org.transmartproject.core.multidimquery
 
+import groovy.transform.Canonical
 import groovy.transform.Immutable
 
 /**
  * Numerical values aggregates
  */
-@Immutable
+@Canonical
```

Comment:
Why?

Reply:
Because of consistency, see [CategoricalValueAggregates](https://github.com/thehyve/transmart-core/blob/dev/transmart-core-api/src/main/groovy/org/transmartproject/core/multidimquery/CategoricalValueAggregates.groovy).